### PR TITLE
[SCB-2267]add registryName api

### PR DESCRIPTION
**Pull request by kaister3**

### CODE REVIEW

File: service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java [hunk 1]
```diff
@@ -253,6 +253,9 @@ public static void executeOnEachServiceRegistry(Consumer<ServiceRegistry> action
 
   public static void addExtraServiceRegistry(ServiceRegistry serviceRegistry) {
     Objects.requireNonNull(serviceRegistry);
+    if (EXTRA_SERVICE_REGISTRIES.containsKey(serviceRegistry.getName())) {
```
Comment (liubao68) [inline service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java:256]: and name can not be the default regitry name

File: service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java [hunk 2]
```diff
@@ -253,6 +253,9 @@ public static void executeOnEachServiceRegistry(Consumer<ServiceRegistry> action
...
+      throw new IllegalArgumentException("Register name duplicated!");
```
Comment (liubao68) [inline service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java:257]: reigstriy name {name} is duplicated, please set different names for each implementation.

Comment (kaister3) [reply service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java:257]: when duplicated, I can only get the new client name, cannot get old client name from EXTRA_SERVICE_REGISTRY. How to set different implmentation?

File: service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java (hunk 1 above)
Comment (kaister3) [reply service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java:256]: getName() here returns custom registry name, not default name

**Review by liubao68 — APPROVED** [review]

**Pull request merged**